### ISSUE TITLE
[pni] product: missing h1 heading

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#4646]
Pages should have an `<h1>` heading ("somewhere") that accessibility tools can use as summary of the page content.

The most obvious choice here is to just mark up the product name with an `<h1>`.
